Sources/VRage.Dedicated/Configurator/ConfigForm.cs

```diff
@@ -68,7 +68,8 @@ private void startButton_Click(object sender, EventArgs e)
             }
             else // Local / Console
             {
-                Process.Start(MyPerServerSettings.GameDSName + ".exe", "-console -ignorelastsession");
+                string[] cmdLine = Environment.CommandLine.Split(null as string[], 2, StringSplitOptions.RemoveEmptyEntries);
+                Process.Start(cmdLine[0], cmdLine[1] + " -console -ignorelastsession");
```

Comment:
I got an exception when trying to run the compiled DS, I reverted these 2 lines to the old line and it works again.
Exception: http://pastebin.com/raw.php?i=BkY8Jka4